server/job.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"github.com/google/uuid"
+	pb "github.com/romanlevin/felek/jobs"
+	"os/exec"
+)
+
+type job struct {
+	id        string
+	cmd       *exec.Cmd
+	stopped   bool
+	exitError string
+	owner     string
+}
+
+func jobStatus(j *job) *pb.JobStatus {
+	c := j.cmd
+	if j.exitError == "" && c.ProcessState != nil {
+		// Job exited without exitError
+		return &pb.JobStatus{
+			Id: &pb.JobID{Value: j.id},
+			JobState: &pb.JobStatus_StoppedJob{
+				StoppedJob: &pb.StoppedJob{
+					Exit: &pb.StoppedJob_ExitCode{
+						ExitCode: int64(c.ProcessState.ExitCode()),
+					},
+					SystemTime: int64(c.ProcessState.SystemTime()),
+					UserTime:   int64(c.ProcessState.UserTime()),
+					Stopped:    j.stopped,
```

Comment:
Is it safe to access the `stopped` field without acquiring a lock?

Reply:
Definitely is not. Fixing as part of moving to per-job locks.